build_scripts/windows/scripts/build.cmd

```diff
@@ -137,18 +143,27 @@ if %errorlevel% neq 0 goto ERROR
 
 pushd %BUILDING_DIR%
 %BUILDING_DIR%\python.exe %REPO_ROOT%\scripts\compact_aaz.py
+if %errorlevel% neq 0 goto ERROR
 %BUILDING_DIR%\python.exe %~dp0\patch_models_v2.py
+if %errorlevel% neq 0 goto ERROR
 %BUILDING_DIR%\python.exe %REPO_ROOT%\scripts\trim_sdk.py
+if %errorlevel% neq 0 goto ERROR
 popd
 
 REM Remove pywin32 help file to reduce size.
 del %BUILDING_DIR%\Lib\site-packages\PyWin32.chm
 
-echo Creating the wbin (Windows binaries) folder that will be added to the path...
-mkdir %BUILDING_DIR%\wbin
-copy %REPO_ROOT%\build_scripts\windows\scripts\az.cmd %BUILDING_DIR%\wbin\
-copy %REPO_ROOT%\build_scripts\windows\scripts\azps.ps1 %BUILDING_DIR%\wbin\
-copy %REPO_ROOT%\build_scripts\windows\scripts\az %BUILDING_DIR%\wbin\
+if "%TARGET%"=="msi" (
+    REM Creating the wbin (Windows binaries) folder that will be added to the path...
```

Comment:
If I use `echo` instead of `REM`, the `(Windows binaries) folder` will be parsed and `folder was unexpected at this time.` error is raised.
Repro script:
``` cmd
echo () ok
if 'kk'=='kk' (
    echo () ok2
)
```
```
() ok
ok2 was unexpected at this time.
```

Reply:
If you truly need to use `echo`, you may quote the sentence:

```cmd
echo "Creating the wbin (Windows binaries) folder that will be added to the path..."
```

Reply:
Exactly. (Generally, `echo` is able to print multiple word without the quote: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/echo)

In this instance, since the line functions more as a comment, so I changed it to `REM`.